src/js/components/Select/Select.js

```diff
@@ -117,6 +120,7 @@ class Select extends Component {
             border={!plain ? 'all' : undefined}
             direction='row'
             justify='between'
+            background={theme.select.backgroundColor}
```

Comment:
I still think this should be `theme.drop.backgroundColor`

Reply:
Now that I understand what you are doing here, sorry I misunderstood for so long :( , can we name this `theme.select.background`? That would allow us to pass through anything that Box's background property would accept, which includes opacity.